.github/workflows/validate_new_quickstarts_bindings.yaml

```diff
@@ -39,17 +39,14 @@ jobs:
       KIND_IMAGE_SHA: sha256:69860bda5563ac81e3c0057d654b5253219618a22ec3a346306239bba8cfa1a6
     strategy:
       matrix: 
-        os: [ubuntu-latest, macos-latest]
+        os: [ubuntu-latest, macos-10.15]
```

Comment:
Is MacOS version pinning part of the solution here?  why?  thank you

Reply:
Yes! Macos latest is not working for docker installation.Hence using the previous mac version